tests/js/spec/views/performance/transactionSummary/suspectSpans.spec.tsx

```diff
@@ -0,0 +1,68 @@
+import {initializeOrg} from 'sentry-test/initializeOrg';
+import {generateSuspectSpansResponse} from 'sentry-test/performance/initializePerformanceData';
+import {act, mountWithTheme, screen} from 'sentry-test/reactTestingLibrary';
+
+import ProjectsStore from 'app/stores/projectsStore';
+import EventView from 'app/utils/discover/eventView';
+import SuspectSpans from 'app/views/performance/transactionSummary/transactionOverview/suspectSpans';
+
+function initializeData({query} = {query: {}}) {
+  const features = ['performance-view', 'performance-suspect-spans-view'];
+  const organization = TestStubs.Organization({
+    features,
+    projects: [TestStubs.Project()],
+  });
+  const initialData = initializeOrg({
+    organization,
+    router: {
+      location: {
+        query: {
+          transaction: 'Test Transaction',
+          project: '1',
+          ...query,
+        },
+      },
+    },
+  });
+  act(() => void ProjectsStore.loadInitialData(initialData.organization.projects));
+  return {
+    ...initialData,
+    eventView: EventView.fromLocation(initialData.router.location),
+  };
+}
+
+describe('SuspectSpans', function () {
+  describe('With Span Data', function () {
+    beforeEach(function () {
+      MockApiClient.addMockResponse({
+        url: '/organizations/org-slug/events-spans-performance/',
+        body: generateSuspectSpansResponse({
+          examples: 1,
+        }),
+      });
+    });
+
+    it('renders basic UI elements', async function () {
```

Comment:
You _might_ want to add some checks for the content of the cells (eg. checking links or the data is correctly passing from your mocked objects above), but you can always add it in a future PR where you are checking a cell content use case.

Reply:
Due to the way tests setup react-router, the links do not have any hrefs, skipping these tests for now.

Reply:
Due to the way tests setup react-router, the links do not have any hrefs, skipping these tests for now.